packages/supermassive/src/__tests__/execute.test.ts

```diff
@@ -301,17 +301,14 @@ async function compareResultsForExecuteWithoutSchema(
   variables: Record<string, unknown> = {},
 ) {
   expect.assertions(1);
-  const fullResolvers = mergeResolvers(
-    resolvers as Resolvers<any, any>,
-    (extractedResolvers as any) as Record<string, Resolver<any, any>>,
-  );
   const document = parse(query);
   const result = await executeWithoutSchema({
     document: addTypesToRequestDocument(schema, document),
     contextValue: {
       models,
     },
-    resolvers: fullResolvers,
+    resolvers: resolvers as BasicResolvers,
+    schemaResolvers: (extractedResolvers as any) as Resolvers,
```

Comment:
Why do we need those type casts?

If we generate typescript, maybe we should annotate generated file to have Resolvers type annotation.